mosaiks.py

```diff
@@ -101,11 +94,11 @@ def load_dataset(url, shuffle_on_load):
 
     logger.info("Loading evaluation tasks")
     eval_task_list: List[EvalTask] = [
-        *[FuelMoistureEval(seeds=[s]) for s in eval_seeds],
-        *[AlgaeBloomsEval(seeds=[s]) for s in eval_seeds],
-        *[CropHarvestEval("Kenya", seeds=[s]) for s in eval_seeds],
-        *[CropHarvestEval("Togo", seeds=[s]) for s in eval_seeds],
-        *[CropHarvestEval("Brazil", seeds=[s]) for s in eval_seeds],
+        FuelMoistureEval(seed=seed),
```

Comment:
So for mosaiks you start a wandb run per seed? ([line 91](https://github.com/nasaharvest/presto/pull/32/files#diff-8943f236dd39782f2e4786a6986d26817c8f66c9d4c11804f17d9f6aeefaf684R91))

Reply:
yes, because each seed yields a different MOSAIKS model.